Allow For Building on Linux

onlycs opened the pull request:
### Resolves

__What Github issue does this resolve (please include link)?__ 
None, I just want to build on Linux.

### Proposed Changes
__Describe what this Pull Request Does__
Allows For Building on Linux. It gave errors on Linux before.

### Reason for Changes

__Explain why these changes should be made__
Because a lot of people want to use Scratch Desktop on Linux.

### Test Coverage

__Please show how you have added tests to cover your changes__
I built a .deb file without errors on Linux (Ubuntu 20.04 64-bit).

Please change the Email if necessary(on package.json), I didn't know what to put there, but it was apparently a requirement on Linux.

onlycs committed:
Allow For Building on Linux (deb)

apple502j commented:
The build broke on Windows...

dragonBall561 commented:
Think I know why. I am using `./node_modules/...#Whatever goes here` for some reason the '.' is doing something weird. Let me see if I can fix it.

onlycs committed:
Hopefully Fixed Windows Build

Linux still works, just reverted back to the old electron-builder path if it was Windows.

dragonBall561 commented:
Confirmed Working, the edited files built on windows.

eMPee584 commented:
@apple502j so would you consider this ready for merging? 
will be great to finally have linux added back to the supported scratch platforms 🤠

apple502j commented:
I'm not a Scratch Team member, fyi.

dragonBall561 commented:
@eMPee584 

I fixed all the problems a long time ago.. This works.

onlycs committed:
Merge branch 'develop' into develop

ezze commented:
Any chance it will be merged?

eMPee584 commented:
.. seems @cwillisf is the sole committer since a year..
Dear Christopher, any objections to this simple build fix? 🤓

dragonBall561 commented:
Idk why this isn't getting merged. All you guys have to do is add Linux as a building target and voila!
